modules/loader.js

```diff
@@ -761,11 +761,10 @@ __private.sync = function(cb) {
 	async.series(
 		{
 			getPeersBefore(seriesCb) {
-				library.logger.debug('Establishing broadhash consensus before sync');
-				return modules.transport.getPeers(
-					{ limit: constants.maxPeers },
-					seriesCb
+				library.logger.debug(
+					`Establishing broadhash consensus before sync: ${modules.peers.getLastConsensus()} %`
```

Comment:
Not gonna call `modules.transport.getPeers` anymore? If not, why call it `getPeersBefore`?

Reply:
True, the loader's steps are now calculating consensus only, so if we decide to keep calculating consensus before and after sync it should be renamed 'getConsensusBefore' and 'calculateConsensusAfter'.